spec.md

```diff
@@ -772,15 +775,21 @@ For success responses, the following fields are defined:
 | Response Field | Type | Description |
 | --- | --- | --- |
 | state* | string | Valid values are `in progress`, `succeeded`, and `failed`. While `"state": "in progress"`, the Platform SHOULD continue polling. A response with `"state": "succeeded"` or `"state": "failed"` MUST cause the Platform to cease polling. |
+| description | string | A user-facing message that can be used to tell the user details about the status of the operation. If present, MUST be a non-empty string. |
```

Comment:
The description field appears twice in this section. Conflict resolution artifact?

Reply:
Thanks. Fixed.

Reply:
Great! Thank you!